lib/cryptoaes/src/main/java/eu/kanade/tachiyomi/lib/cryptoaes/CryptoAES.kt

```diff
@@ -130,4 +131,28 @@ object CryptoAES {
             Arrays.fill(generatedData, 0.toByte())
         }
     }
+
+    /**
+     * Decrypt a file using AES CTR mode
+     *
+     * @param encryptedData the encrypted data
+     * @param keyBytes the key to decrypt the data
+     * @return the decrypted data
+     */
+
+    fun decryptFile(encryptedData: ByteArray, keyBytes: ByteArray, hashCipher: String = HASH_CIPHER): ByteArray {
```

Comment:
implement this directly in the source

Reply:
> implement this directly in the source

I disagree with this change but I will make it